### PR TITLE
Fixes needed to generate ISOs from image builder

### PR DESCRIPTION
**Pull request by wjt**
https://phabricator.endlessm.com/T13559

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -252,7 +252,13 @@ Package: eos-tech-support
 Architecture: all
 Depends: ${misc:Depends}, ${eos:Depends}
 Description: EndlessOS Technical Support Tools
- This is a collection technical support tools that are installed by default
+ This is a collection of technical support tools that are installed by default
  with the EndlessOS operating system.  They are not required as part of the
  normal user experience but are relatively small and helpful for advanced
  troubleshooting.
+
+Package: eos-tech-support-iso
```
Comment (wjt) [inline debian/control]: This package doesn't actually get built, and I'm not sure why…

Comment (dbnicholson) [inline debian/control]: So, the point here is that we want to pull the dependencies into the image builder but not into the ostree?

Comment (wjt) [inline debian/control]: Exactly. We could of course choose to include these dependencies in the ostree – they're not *that* big – but I'd rather that question be separate from "can the image builder make ISOs".

Comment (dbnicholson) [inline debian/control]: That's OK, I agree. It's not ideal (if we're shipping tools, we should actually provide the things they need to run), but in the short term this makes sense. I just wanted to make sure I understood the purpose of the separate metapackage.

**Pull request merged**